### PR TITLE
Addressing review comments for BD and Coverity examples

### PR DESCRIPTION
**Pull request by spurohitsynopsys**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ Before configuring Synopsys Security Scan into your azure pipeline, note the fol
 
 ## Synopsys Security Scan - Polaris
 
-Synopsys Security Scan Extension available in the Azure DevOps Marketplace is the recommended solution for integrating Polaris into Azure pipeline.
+Synopsys Security Scan Extension available in the Azure DevOps Marketplace is the recommended solution for integrating Polaris into Azure pipeline. The extension will download the Synopsys Bridge CLI, execute a scan, and offers post-scan features such as break-the-build quality gates.
```
Comment (sig-tithi) [inline README.md:47]: change offer to offers or make the sentence in present tense. The Extension downloads Synopsys Bridge CLI.  the before synopsys bridge is not needed.

File: README.md [hunk 2]
```diff
@@ -170,6 +171,7 @@ steps:
 - task: SynopsysSecurityScan@1.0.0
   displayName: 'Black Duck Full Scan'
   condition: not(eq(variables['Build.Reason'], 'PullRequest'))
+  ### Use below configurations to set specific detect environment variables
   env:
     DETECT_PROJECT_NAME: $(Build.Repository.Name)
```
Comment (sig-tithi) [inline README.md:176]: detect params for action madhu asked to remove so we can keep that consistent if he suggests for this one too.

File: README.md [hunk 3]
```diff
@@ -44,9 +44,9 @@ Before configuring Synopsys Security Scan into your azure pipeline, note the fol
 
 ## Synopsys Security Scan - Polaris
 
-Synopsys Security Scan Extension available in the Azure DevOps Marketplace is the recommended solution for integrating Polaris into Azure pipeline.
+Synopsys Security Scan Extension available in the Azure DevOps Marketplace is the recommended solution for integrating Polaris into Azure pipeline. The extension will download the Synopsys Bridge CLI, execute a scan, and offers post-scan features such as break-the-build quality gates.
 
-Here's an example piepline for Polaris scan using the Synopsys Synopsys Security Scan:
+Here's an example pipeline for Polaris scan using the Synopsys Synopsys Security Scan:
```
Comment (sig-tithi) [inline README.md:49]: in polari exa,ple we have used '' in right side of parameter values. and assessment_types are "". is that fine?

Comment (spurohitsynopsys) [reply README.md:49]: yes

File: README.md (hunk 1 above)
Comment (spurohitsynopsys) [reply README.md:47]: we can have since download CLI is also part of our scan process

File: README.md [hunk 4]
```diff
@@ -123,7 +124,7 @@ steps:
     BRIDGE_COVERITY_CONNECT_USER_NAME: $(COVERITY_USER)
     BRIDGE_COVERITY_CONNECT_USER_PASSWORD: $(COVERITY_PASSPHRASE)
```
Comment (sig-tithi) [inline README.md:125]: in all 3 workflow examples few places we are having '' on right side of variable value placeholder rest don't can we make it consistent?

Comment (spurohitsynopsys) [reply README.md:125]: removed

**Review by sig-tithi — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (spurohitsynopsys) [reply README.md:176]: updated

**Pull request merged**